src/service/api.ts

```diff
@@ -1,19 +1,10 @@
 import axios from 'axios';
 
-const BASE_URL = "https://randomuser.me/api/?results="
+const BASE_URL = "https://randomuser.me/api"
```

Comment:
add this to the `.env` file so it's configurable! also, create a .env.sample file

Reply:
done

Reply:
added